android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/manager/FetchDownloadManager.kt

```diff
@@ -71,6 +76,13 @@ class FetchDownloadManager<T : AbstractFetchDownloadService>(
         }
         validatePermissionGranted(applicationContext)
 
+        if (SDK_INT >= Build.VERSION_CODES.TIRAMISU &&
+            getTargetSDKVersion() >= Build.VERSION_CODES.TIRAMISU &&
+            !NotificationManagerCompat.from(applicationContext).areNotificationsEnabled()
```

Comment:
⚠️ This could throw a `RemoteException`.  

As it's already being checked in `notificationsDelegate.requestNotificationPermission()` maybe we can call that already without the conditions.

Reply:
Removed the additional check.